### DISCUSSION TITLE
配置缓存缓存redis之后报500服务器错误

### DISCUSSION THREAD

**Discussion by SerMs** [lsky-org/lsky-pro#637] [问答]
配置缓存缓存redis之后报500服务器错误
![image](https://user-images.githubusercontent.com/67584847/226539064-e9754dd0-cf99-43c5-8f97-ef508dda06ea.png)
![image](https://user-images.githubusercontent.com/67584847/226539163-9a06e071-eb10-4895-aba8-08deb10ccb44.png)
另外一个.env的文件也改过，也一样报错
![image](https://user-images.githubusercontent.com/67584847/226539293-9921793c-1bd7-43cb-8356-99f1b2f63137.png)

**Comment by SerMs** [comment]
![image](https://user-images.githubusercontent.com/67584847/226540094-3864c9a3-1b21-4900-893f-d99d16f0e4b3.png)

**Comment by 0xxb** [comment]
修改 `APP_DEBUG=true` 可以查看详细的错误信息。

**Comment by SerMs** [comment]
![image](https://user-images.githubusercontent.com/67584847/226779875-17766daa-1a7c-4b78-9e52-8619e05c8b63.png)
![image](https://user-images.githubusercontent.com/67584847/226779911-983919fc-740a-4351-9968-704398d0ac41.png)
![image](https://user-images.githubusercontent.com/67584847/226779944-105a0e49-ada4-48a7-8c69-91e1e1cde481.png)
麻烦帮我看一下

**Reply by 0xxb** [reply]
未安装 redis 拓展，先给 PHP 安装 redis 拓展，如果已经安装了，尝试重启 PHP。
